tests/spec/cmdline_spec.rb

```diff
@@ -74,4 +74,15 @@
       expect(result).to be true
     end
   end
+
+  describe '.changed_since' do
+    it 'gitarro should see at least PR #30 changed in the last 60 seconds' do
+      context = 'changed-since'
+      desc = 'changed-since'
+      @comment = @rgit.create_comment(@pr, "gitarro rerun #{context} !!!")
```

Comment:
you need here to target explicetly the PR number 30, you will write on the first pr open which is not the 30.
i